index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <!-- CSS only -->
+  <link rel="stylesheet" href="./assets/css/style.css" />
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi" crossorigin="anonymous" />
+  <title>Infinity Web Development</title>
+</head>
+
+<body>
+  <header class="top-header">
+    <div class="nav1 bg-dark d-none d-md-flex justify-content-end align-items-center gap-3 text-white">
+      <a href=""><i class="fa-brands">&#xf09a;</i></a>
+      <a href=""><i class="fa-brands">&#xf099;</i></a>
+      <a href="">English</a>
+      <a href="">My page</a>
+      <a href="">Logout</a>
+    </div>
+    <nav class="nav-bar">
+      <div class="logo"><a href="" class="d-flex align-items-center text-decoration-none text-black">
+          <lord-icon src="https://cdn.lordicon.com/dfxesbyu.json" trigger="loop" delay="800"
+            colors="primary:#242424,secondary:#ec5242" style="width:45px;height:45px">
+          </lord-icon><span>Infinity</span>
+        </a></div>
+      <ul class="menu">
+        <li><a class="menu-item close"><img src="./assets/images/close.png" alt="close icon"></a></li>
+        <li class="menu-item"><a href="./index.html">Home</a></li>
+        <li class="menu-item"><a href="./about-page.html">About</a></li>
+      </ul>
+    </nav>
+  </header>
+  <section class="header">
+    <a> <img class="humbarger" src="./assets/images/icons8-menu-50.png" width="40px" alt="Menu icon"></a>
+    <div class="headline">
+      <h1 class="h2">Hello, Sharing world</h1>
+      <h2 class="title">Innovative, Reliable, and Effective Web Development</h2>
```

Comment:
- [ ] I think it would be better if you update the styling of this title to match the template design e.g.: text color, font size, font weight,  text-transform,...
![Screenshot from 2022-11-24 00-10-02](https://user-images.githubusercontent.com/83042742/203654399-7f665de8-9db6-44ae-bb66-2f6d87cb3b4a.png)